examples/with-sequelize/README.md

```diff
@@ -0,0 +1,51 @@
+# Razzle Sequelize Example
+
+## How to use
+
+Download the example [or clone the whole project](https://github.com/jaredpalmer/razzle.git):
+
+```bash
+curl https://codeload.github.com/jaredpalmer/razzle/tar.gz/master | tar -xz --strip=2 razzle-master/examples/with-sequelize
+cd with-sequelize
+```
+
+### Database server
+
+The example comes with a `docker` container to bootstrap a database server. For this to work you should have `docker` installed. If you already have `mysql` installed and want to use that server instead of `docker`, create a `.env` file in the `with-sequelize` root, with the following values:
```

Comment:
I like this. I think we should rework it to assume that you connect to sqllite with just dotenv vars at first. It’s a sequelize example after all. then at the bottom let’s document how to spin up local docker mysql and and postgres as well as just installing these with brew?

Reply:
Actually screw sqllite, I’m fine with it as mysql or postgres  as long as we can use a locally installed dB in the example (without docker)